examples/draw_M_z_from_hmf.ipynb

```diff
@@ -0,0 +1,863 @@
+    "    \n",
+    "    # compute the weights to be used to bluid the DeltaSigma radial profiles\n",
+    "    cl.compute_galaxy_weights(use_pdz = True,\n",
+    "    use_shape_noise = True, shape_component1 = 'e1', shape_component2 = 'e2', \n",
+    "    use_shape_error = True, shape_component1_err = 'e_err', shape_component2_err = 'e_err', \n",
+    "    weight_name = 'w_ls', cosmo = cosmo_clmm, is_deltasigma = True, add = True)\n",
+    "\n",
+    "    # append the cluster in the list\n",
+    "    gclist.append(cl)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b82f03d9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cl.galcat.columns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f0e4bf7",
+   "metadata": {},
+   "source": [
+    "## Creating ClusterEnsemble object and estimation of individual excess surface density profiles\n",
+    "From the galaxy cluster object list `gclist`, we instantiate a cluster ensemble object `clusterensemble`. This instantiation step uses\n",
+    " - the individual galaxy input $\\Delta\\Sigma_+$ and $\\Delta\\Sigma_{\\times}$ values (computed in the previous step, `DS_{t,x}`)\n",
+    " - the corresponding individual input weights `w_ls` (computed in the previous step)\n",
+    "\n",
+    "to compute\n",
+    "\n",
+    "- the output tangential `DS_t` and cross signal `DS_x` binned profiles (where the binning is controlled by `bins`)\n",
+    "- the associated profile weights `W_l` (that will be used to compute the stacked profile)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ddd5aff1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ensemble_id = 1\n",
+    "names = ['id', 'ra', 'dec', 'z', 'radius', 'gt', 'gx', 'W_l']\n",
+    "bins = np.logspace(np.log10(0.3),np.log10(5),10)\n",
+    "clusterensemble = ClusterEnsemble(ensemble_id, gclist, \n",
+    "                                  tan_component_in='DS_t', cross_component_in='DS_x', \n",
+    "                                  tan_component_out='DS_t', cross_component_out='DS_x', \n",
+    "                                  weights_in = 'w_ls', weights_out = 'W_l', \n",
+    "                                  bins=bins, bin_units='Mpc', cosmo=cosmo_clmm)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9091de7c",
+   "metadata": {},
+   "source": [
+    "## Stacked profile of the cluster ensemble\n",
+    "The stacked radial profile of the ensemble is then obtained as "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b6162ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "clusterensemble.make_stacked_radial_profile(tan_component='DS_t', cross_component='DS_x')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "771c1382",
+   "metadata": {},
+   "source": [
+    "## Covariance (Bootstrap, sample, Jackknife) of the stack between radial bins\n",
```

Comment:
I don't know if we are keeping the covariance analysis on this notebook.

Reply:
I've left it for the time being, but maybe we can just use one covariance and not compute the three. Let's see with whoever reviews the PR.